pkg/cmd/admin/router/router.go

```diff
@@ -276,7 +276,7 @@ func NewCmdRouter(f *clientcmd.Factory, parentName, name string, out, errout io.
 	cmd.Flags().StringVar(&cfg.ForceSubdomain, "force-subdomain", "", "A router path format to force on all routes used by this router (will ignore the route host value)")
 	cmd.Flags().StringVar(&cfg.ImageTemplate.Format, "images", cfg.ImageTemplate.Format, "The image to base this router on - ${component} will be replaced with --type")
 	cmd.Flags().BoolVar(&cfg.ImageTemplate.Latest, "latest-images", cfg.ImageTemplate.Latest, "If true, attempt to use the latest images for the router instead of the latest release.")
-	cmd.Flags().StringVar(&cfg.Ports, "ports", cfg.Ports, "A comma delimited list of ports or port pairs to expose on the router pod. The default is set for HAProxy. Port pairs are applied to the service and to host ports (if specified).")
```

Comment:
It appears in the router pod containerPort and hostPort:
    ports:
    - containerPort: 1080
      hostPort: 1080
      protocol: TCP
It also appears in the service:
exampleport         172.30.138.192   <none>        1080/TCP,1935/TCP,1936/TCP               4m
How do you want to phrase this?